ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -58,6 +60,7 @@ type ssh_daemon_options_type = {
     "KerberosGetAFSToken"               ? ssh_yesnostring
     "KerberosOrLocalPasswd"             ? ssh_yesnostring
     "KerberosTgtPassing"                ? ssh_yesnostring
+    "KerberosTicketAuthentication"      ? ssh_yesnostring
     "KerberosTicketCleanup"             ? ssh_yesnostring
     "KeyRegenerationInterval"           ? long
     "ListenAddress"                     ? string
```

Comment:
as an example, this has to beocme 
```pan
@{ListenAddress, one per line}
ListenAddress ? type_hostport[]
```
the docstring `@{}` is optional, but since this change in partiticular is backwards incompatible,  and you need to add a 
```pan
include 'pan/types';
```
to make sure the `type_hostport` is usable